README.md

```diff
@@ -0,0 +1,25 @@
+Terraform Controller
+====================
+
+Introduction
+------------
+
+This is a custom Kubernetes controller designed to run in the kube-system 
+namespace. It watches for changes on incoming CRD's and launches Jobs to 
+create/update/destroy infrastructure using Terraform.
+
+It is built with [Kubebuilder][kubebuilder], with full documentation found 
+[here][kubebuilder-book].
+
+[kubebuilder]: https://github.com/kubernetes-sigs/kubebuilder
+[kubebuilder-book]: https://book.kubebuilder.io/
+
+Running Locally
```

Comment:
I copied your readme for the ssh-controller readme PR, so apparently I'm pretty ok with this as is :)

That being said, should we include any prerequisites here? Go version? Maybe a link to dep? Maybe that stuff is too basic for a Go project?

Reply:
Maybe I explicitly point to the kubebuilder github for system requirements?

Reply:
Oh good point. Sounds good.